webapp/app/model/form_data.py

```diff
@@ -354,13 +354,13 @@ def delete_if_show_person_b(cls, v, values):
         return v
     
     @validator('person_a_requests_pauschbetrag')
-    def delete_if_person_a_has_no_disability(cls, v, values):
+    def required_if_person_a_has_disability(cls, v, values):
         if values.get('person_a_has_disability') == "yes":
             return v
         return None
     
     @validator('person_b_requests_pauschbetrag')
-    def delete_if_person_b_has_no_disability(cls, v, values):
+    def required_if_person_b_has_disability(cls, v, values):
```

Comment:
Actually, it was correct before. As you can see, the value is either returned as is or as None in case the person does not have a disability. Hence: `delete_if_person_a_has_no_disability`.

Reply:
Done